R/pkg/R/SQLContext.R

```diff
@@ -75,8 +75,8 @@ infer_type <- function(x) {
 #'\dontrun{
 #' sc <- sparkR.init()
 #' sqlContext <- sparkRSQL.init(sc)
-#' rdd <- lapply(parallelize(sc, 1:10), function(x) list(a=x, b=as.character(x)))
-#' df <- createDataFrame(sqlContext, rdd)
+#' df1 <- as.DataFrame(sqlContext, iris)
```

Comment:
I see.

It's OK the remove the example of RDD API. What I meant is that we have examples for both createDataFrame() and as.DataFrame(). I see that NarineK added examples, it is OK now.